### PR TITLE
new Tags chapter and Custom Fields chapter 

### PR DESCRIPTION
**Pull request by ghost**
This includes new Tags chapter and Custom Fields chapter. Also a little bit of Tenants chapter. 

I also had to fix all the image references to include the relative portion of the path name. Hope I did it right!

### CODE REVIEW

**Comment by sbrossie** [comment]
@FlowWriter Could you attach the PDF - if you have it?

**Comment by reshmabidikar** [comment]
Hi @FlowWriter, I see that there is a [tags.pdf](https://github.com/FlowWriter/killbill-docs/blob/kaui-only-9/userguide/kaui/includes/tags.pdf) but it appears to be different from [tags.adoc](https://github.com/FlowWriter/killbill-docs/blob/kaui-only-9/userguide/kaui/includes/tags.adoc). So, perhaps `tags.pdf` is an older version and needs to be updated? Secondly, the pdf file for [custom-fields.adoc](https://github.com/FlowWriter/killbill-docs/blob/kaui-only-9/userguide/kaui/includes/custom-fields.adoc) is missing, could you please add that too?

**Comment by ghost** [comment]
@sbrossie and @reshmabidikar - Here is the PDF of the entire document. You only need to check the Tags and Custom Fields chapters.
[Kaui_Guide_Draft.pdf](https://github.com/killbill/killbill-docs/files/8254025/Kaui_Guide_Draft.pdf)

**Comment by sbrossie** [comment]
@FlowWriter Looking great, I just have one comment/question: On Page 76, there is a screenshot for the Account#tags, but you are describing tags associated with an invoice - should it be removed?

File: userguide/kaui/includes/tags.adoc [hunk 1]
```diff
@@ -1 +1,127 @@
...
+
+[NOTE]
+*Note:* You can use the API to attach tags to even more object types, such as payments and subscriptions. For more information, see the "Tags" section of the relevant object type section (for example, Payments) in the https://killbill.github.io/slate/[API Reference].
+
+==== Types of Tags
+
+Kill Bill has two types of tags:
+
+* *Control tags*&#8212;Also referred to as _system tags_, these tags modify the behavior of the system. In Kaui, you can view and attach these tags, but you cannot add, change, or delete the underlying <<Add Tag Definition, tag definition>>. Kill Bill includes several control tags; see the API documentation for https://killbill.github.io/slate/#tag[a list].
+
```
Comment (reshmabidikar) [inline userguide/kaui/includes/tags.adoc:15]: Looks like the [Account Tags](https://killbill.github.io/slate/#account-tags) section lists all the tags. I will move this to the [Tag](https://killbill.github.io/slate/#tag) section in the slate docs.

File: userguide/kaui/includes/tags.adoc [hunk 2]
```diff
@@ -1 +1,127 @@
-// For the tags, it would make sense to touch on the 'system' tags versus user tags, the former (Uppercase) having a strong effect on the behavior of the system, while the latter (lowercase) having NO effect on the behavior of the system.
+=== Overview
+
+In Kaui, you can apply one or more tags to accounts and invoices, a tag being a single value. Tags in Kill Bill can be used to convey information about an object type. A tag can just provide information about an object (user tag) or it can actually affect how the object is handled in Kill Bill (control tag).
+
+This section explains how to attach tags to accounts and invoices. It also describes the steps for creating the tag definitions on which a tag is based.
+
+[NOTE]
+*Note:* You can use the API to attach tags to even more object types, such as payments and subscriptions. For more information, see the "Tags" section of the relevant object type section (for example, Payments) in the https://killbill.github.io/slate/[API Reference].
+
```
Comment (reshmabidikar) [inline userguide/kaui/includes/tags.adoc:9]: Kaui also allows adding a user tag to a subscription. See attached screenshot.

Subscription_Tag

File: userguide/kaui/includes/tags.adoc [hunk 3]
```diff
@@ -1 +1,127 @@
...
+[TIP]
+*Tip:* On the Tags page, you can open the object (account, invoice, etc.) by clicking the link in the Object ID column.
+
+To see any tags assigned to a specific account (or invoice):
+
+. Open the account on the Account page.
+. Click *Tags* on the sub-menu.
+
+Kaui opens the Tags page and displays only the tags assigned to that account or its related invoice(s).
+
```
Comment (reshmabidikar) [inline userguide/kaui/includes/tags.adoc:65]: Kaui also displays tags associated with subscriptions (if any).

File: userguide/kaui/includes/custom-fields.adoc [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+. At the top of the screen, click the tag icon and select *Custom Fields*.
++
++image::../assets/img/kaui/TagDropdown-CustomFields.png[width=150,align="center"]
++
+Kaui displays the Custom Fields page:
++
+image::../assets/img/kaui/CustomFields-Populated.png[width=850,align="center"]
++
+. On the custom field row, click *Destroy*.
```
Comment (reshmabidikar) [inline userguide/kaui/includes/custom-fields.adoc]: Is this correct? I do not see a **Destroy** link in the custom field row.

**Review by reshmabidikar — COMMENTED** [review]
Looks good, I've added a few minor comments

**Comment by ghost** [comment]
> Looking great, I just have one comment/question: On Page 76, there is a screenshot for the Account#tags, but you are describing tags associated with an invoice - should it be removed?

@sbrossie - In the beginning, I explain that the steps are for applying a tag to accounts _and_ invoices. I can edit the steps to be more clear.

File: userguide/kaui/includes/tags.adoc (hunk 2 above)
Comment (ghost) [reply userguide/kaui/includes/tags.adoc:9]: Thanks, Reshma - I will add that info to the guide.

File: userguide/kaui/includes/custom-fields.adoc (hunk 4 above)
Comment (ghost) [reply userguide/kaui/includes/custom-fields.adoc]: Looks like I used the image ref syntax incorrectly or my graphic is missing. I will fix.

Comment (ghost) [reply userguide/kaui/includes/custom-fields.adoc]: @reshmabidikar - Stéphane said that functionality appears to be missing from Kaui. You don't see "Destroy" when you're logged in, do you?

**Comment by ghost** [comment]
Uploading latest PDF
[Kaui_Guide_Draft.pdf](https://github.com/killbill/killbill-docs/files/8307387/Kaui_Guide_Draft.pdf)

File: userguide/kaui/includes/tags.adoc (hunk 2 above)
Comment (ghost) [reply userguide/kaui/includes/tags.adoc:9]: See page 76 of PDF.

File: userguide/kaui/includes/tags.adoc (hunk 3 above)
Comment (ghost) [reply userguide/kaui/includes/tags.adoc:65]: I edited the section to indicate the process was the same across accounts, invoices, and subscriptions.

File: userguide/kaui/includes/custom-fields.adoc (hunk 4 above)
Comment (ghost) [reply userguide/kaui/includes/custom-fields.adoc]: I have removed the "deleting" section from the Custom Fields chapter.

**Comment by ghost** [comment]
I have uploaded the latest PDF, so I'm hoping this is ready to merge. :-)

**Comment by sbrossie** [comment]
Merging...  @reshmabidikar If somehow if you feel like your comments were not correctly addressed, chime in, we can always open a new PR.

**Pull request merged**